operator/v1/types_csi_cluster_driver.go

```diff
@@ -61,6 +73,16 @@ const (
 // ClusterCSIDriverSpec is the desired behavior of CSI driver operator
 type ClusterCSIDriverSpec struct {
 	OperatorSpec `json:",inline"`
+	// StorageClassState determines if CSI operator should create and sync storage classes.
```

Comment:
Manage feels like it would be a better word here in context.

You should also add a list of valid values as they are otherwise not included in swagger docs.

```suggestion
	// StorageClassState determines if CSI operator should create and manage storage classes.
	// Valid values are Managed, Unmanaged, Removed and omitted.
```